### PR TITLE
[NUI] Add Slider discrete value properties

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- If the app sets `IsDiscrete` as true,
 the thumb snaps to the nearest discrete value when the user drags the thumb or taps.

### API Changes ###
- N/A
-

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Slider.cs [hunk 1]
```diff
@@ -968,6 +968,48 @@ public string ValueIndicatorUrl
...
         }
 
+        /// <summary>
+        /// Flag to decide whether the thumb snap to the nearest discrete value when the user drags the thumb or taps.
+        ///
+        /// The default value is false.
+        /// </summary>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsDiscrete
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/Slider.cs:978]: How about this? it looks simple.
https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/classes-and-structs/properties#auto-implemented-properties

remove `isDiscrete` and change like this =>
`public bool IsDiscrete { get; set; }`

please consider this from next time.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Comment by dongsug-song** [comment]
@Seoyeon2Kim 
Hello, 
please do "update branch" and please do "rebase and merge" for yourself.~
thank you.

File: src/Tizen.NUI.Components/Controls/Slider.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Slider.cs:978]: Thanks for your review.
Could you let me know how I can initialize `IsDiscrete` property as `false` value?
I'll update it right away.

Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Slider.cs:978]: Thanks for your review.
I'll update it right away.

**Pull request merged**